src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -157,7 +158,8 @@ public static IEnumerable<ClassMirror> GetAllTypes(Core core)
                 // if not used in the language: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6752
                 return core.ClassTable.ClassNodes.
                     Where(x => !CoreUtils.StartsWithSingleUnderscore(x.Name)
-                    && x.Name != DSDefinitions.Keyword.PointerReserved).
+                    && x.Name != DSDefinitions.Keyword.PointerReserved
+                    && !x.HasDSDerivedClass).
```

Comment:
So we are not returning class nodes from FFI if it has derived class? I don't think we need to do this, you can use the attribute to hide the base class in library and autocompletion.